src/Indexer.php

```diff
@@ -174,6 +185,8 @@ public function index(): Promise
                     if ($cacheKey !== null) {
                         $this->client->window->logMessage(MessageType::INFO, "Storing $packageKey in cache");
                         $this->cache->set($cacheKey, $index);
+                    } else {
+                        $this->client->window->logMessage(MessageType::INFO, "Cannot cache $packageName, cache key was null. Either you are using a 'dev-' version or your composer.lock is missing references.");
```

Comment:
Use a warning and shorten text to `Could not compute cache key for $packageName`